contracts/README.md

```diff
@@ -0,0 +1,43 @@
+# Contracts
+
+This directory implements the IBC Eureka protocol stack in Solidity. Most entrypoints are upgradeable and deploy per-client/per-channel components through beacons so packet handling, tokenization, and message execution can be upgraded independently.
```

Comment:
- IBC v2 does not have channels.
- Do we call it IBC v2 protocol or IBC Eureka protocol

```suggestion
This directory implements the IBC v2 protocol stack in Solidity. Most entrypoints are upgradeable and deploy per-client/per-user components through beacons so packet handling, tokenization, and message execution can be upgraded independently.
```